mappers/bluetooth_mapper/action_manager/action_manager.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package action_manager
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/kubeedge/kubeedge/common/beehive/pkg/common/log"
```

Comment:
Can we use golang inbuilt log instead of beehiveLog which uses go-chassis and service comb loggers..... It will reduce the Binary size i guess.
Thoughts @m1093782566 ?

Reply:
I had used this log library as this has been used throughout the KubeEdge code and i wanted to keep the mapper in line with it, also this implementation of logging service also differentiates error logs, info logs etc. which is quite helpful. The binary size for this mapper is also not too big (around 10mb), so do you guys @sids-b , @rohitsardesai83 , @m1093782566 feel would it be worth changing it to the default implementation of log ?

Reply:
I guess all logging services support DEBUG, INFO, WARN, ERROR logs ?

Reply:
Can use glog, We have already vendored it

Reply:
The default implementation of go does not support DEBUG, INFO, WARN, ERROR logs, but as suggested, i have used glog :)